### PR TITLE
[CI] running small tests in centos7

### PR DESCRIPTION
**Pull request by ddiezrod**
Just that.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -184,3 +184,9 @@ jobs:
       run: |
         cp .github/workflows/temp_centos_configure.sh temp_centos_configure.sh
         bash temp_centos_configure.sh
+
+    - name: Running small tests
+      run: |
+        export PYTHONPATH=${PYTHONPATH}:${GITHUB_WORKSPACE}/bin/Release
+        export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${GITHUB_WORKSPACE}/bin/Release/libs
+        python3 kratos/python_scripts/run_tests.py -l small -c python3
```
Comment (philbucher) [inline .github/workflows/ci.yml]:
```suggestion
        python3.5 kratos/python_scripts/run_tests.py -l small -c python3.5
```

from experience :D
=> https://github.com/KratosMultiphysics/CoSimIO/pull/151/commits/01596d60a6d6f4a5070d4a99b2d7da77d20297c5

Comment (ddiezrod) [reply .github/workflows/ci.yml]: thanks ;)

**Comment by philbucher** [comment]
seems like there are a few skiptest missing

**Comment by philbucher** [comment]
hm no clue what the error is now

File: kratos/processes/calculate_discontinuous_distance_to_skin_process.cpp [hunk 2]
```diff
@@ -283,7 +283,7 @@ namespace Kratos
         // Initialize cut edges vectors and points arrays
         unsigned int n_cut_edges = 0;
         rIntersectionPointsArray.clear();
-        array_1d<unsigned int, mNumEdges> cut_edges_vector = ZeroVector(mNumEdges);
+        array_1d<unsigned int, mNumEdges> cut_edges_vector(mNumEdges, 0);
```
Comment (philbucher) [inline kratos/processes/calculate_discontinuous_distance_to_skin_process.cpp:286]: ouch, what a nasty conversion problem
Interesting this didn't show up before

did you have this problem in the Altair CI?

https://github.com/KratosMultiphysics/Kratos/blob/284902a7f196d4c19df9fda53d7bf273f9914016/kratos/includes/ublas_interface.h#L80

**Review by philbucher — APPROVED** [review]
ok if it works

File: kratos/processes/calculate_discontinuous_distance_to_skin_process.cpp (hunk 2 above)
Comment (ddiezrod) [reply kratos/processes/calculate_discontinuous_distance_to_skin_process.cpp:286]: Actually no, in Altair ci we are only compiling centos in Release, where the check is not being done.

**Comment by ddiezrod** [comment]
Thanks @philbucher

**Comment by philbucher** [comment]
are you done now or will you also enable MPI?

**Comment by philbucher** [comment]
> are you done now or will you also enable MPI?

ping @ddiezrod

**Comment by ddiezrod** [comment]
> > are you done now or will you also enable MPI?
> 
> ping @ddiezrod

I guess it should be done at some point, though it is not in my immediate plans.

**Pull request merged**